### DISCUSSION TITLE
Missing OS and HW pre-requisites

### DISCUSSION THREAD

**Discussion by TobiW47** [catchthatrabbit/coreminer#69] [Operating System]
I am missing a list of hardware and operating system requirements, because I am currently stuck analyzing a 0 KH/sec issue on Ubuntu 20 LTS

**Comment by rastislavcore** [comment]
Please, provide your system.
There is a limitation described here: https://github.com/catchthatrabbit/coreminer#limitations
Ubuntu 20.04 should work. Please, provide your spec.
